packages/playground/src/App.js

```diff
@@ -94,7 +95,7 @@ export default class extends React.Component {
         domain,
         fetch,
         token: directLineToken,
-        webSocket: webSocket === 'true' || +webSocket
+        webSocket: webSocket === 'true' || !!+webSocket
```

Comment:
Could just be `Boolean(webSocket)`

Reply:
Today, it is either `?ws=true`, `?ws=0` or `?ws=1`. Otherwise, if we are treating it as boolean, I think we could just do `webSocket !== 'false'` (non-`true`/`false`, will by default turns to `true`).

Reply:
I specifically mean `!!+webSocket` can be `Boolean(webSocket)`